api/src/certification/session-management/application/supervise-controller.js

```diff
@@ -1,9 +1,13 @@
+import { getI18n } from '../../../shared/infrastructure/i18n/i18n.js';
 import { usecases } from '../domain/usecases/index.js';
 
 const supervise = async function (request, h) {
   const { 'supervisor-password': invigilatorPassword, 'session-id': sessionId } = request.payload.data.attributes;
   const { userId } = request.auth.credentials;
-  await usecases.superviseSession({ sessionId, userId, invigilatorPassword });
+  const locale = request.query?.lang;
+  const i18n = getI18n(locale);
+
+  await usecases.superviseSession({ sessionId, userId, invigilatorPassword, i18n });
```

Comment:
L’utilitaire `getUserLocale` sait récupérer au mieux la locale de l’utilisateur dans toutes les configurations possibles :

```suggestion
  import { getUserLocale } from '../../../shared/infrastructure/utils/request-response-utils.js';
  const locale = getUserLocale(request);

  await usecases.superviseSession({ sessionId, userId, invigilatorPassword, locale });
```

Reply:
Merci pour l'info ! Avec ton retour sur le fait d'envoyer la date en meta de l'erreur, je vais enlever l'usage du intl ^^